M2/Macaulay2/m2/help.m2

```diff
@@ -251,11 +251,15 @@ documentationValue(Symbol, Package)         := (S, pkg) -> if pkg =!= Core then
 	    star := IMG { "src" => replace("PKG", "Style",currentLayout#"package") | "GoldStar.png", "alt" => "a gold star"};
 	    commit := concatenate(
 		"https://github.com/Macaulay2/M2/blob/",
-		cert#"release at publication", "/M2/Macaulay2/packages/", fn);
+		cert#"release at publication", "/M2/Macaulay2/packages/",
+		(cert#"legacy name" ?? pkg#"pkgname") | ".m2");
```

Comment:
Since we also have the published code URI, what's the point of also linking to an old version on git that may or may not work?

Reply:
Good point.  And for that matter, do we even have a strong reason to link directly to the JSAG version of the code?  It's linked from the article page.

Reply:
I don't mind either way on that.

Reply:
Ok -- I think I'll keep the JSAG code link so at least we have one link to the legacy code but remove the commit info.  It will simplify certifying packages -- finding the exact commit that matches the JSAG version is an annoying step in the release process that I wouldn't mind doing away with.  :)

Reply:
I would say hold off on completely deleting the hash from existing ones, just in case we had a use for it later, but feel free to not populate new ones until it's needed?